### PR TITLE
Update multi-data-centers-in-one-city-deployment.md

### PR DESCRIPTION
**Pull request by rleungx**
### What is changed, added or deleted? (Required)

Remove RTO concept in multi-data-centers-in-one-city-deployment.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: multi-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Raft 是一种分布式一致性算法，在 TiDB 集群的多种组件中，PD
 
 - 所有数据的副本分布在三个数据中心，具备高可用和容灾能力
 - 任何一个数据中心失效后，不会产生任何数据丢失 (RPO = 0)
```
Comment (TomShawn) [inline multi-data-centers-in-one-city-deployment.md:40]: Should this also be removed?
```suggestion
- 任何一个数据中心失效后，不会产生任何数据丢失
```

Comment (rleungx) [reply multi-data-centers-in-one-city-deployment.md:40]: No, just keep it.

**Pull request merged**